.env.example

```
@@ -4,6 +4,10 @@ NODE_ENV=production
 # Site URL (used for emails)
 SITE_URL=http://127.0.0.1
 
+# Websockets URL
+# Leave empty to use SITE_URL
+WS_URL=
```

```suggestion
#WS_URL=
```
See https://github.com/LearningLocker/learninglocker/pull/1470/files#r385788242